network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_reporter_page.html

```diff
@@ -0,0 +1,245 @@
+{% extends "./youtube_regrets_base.html" %}
+{% load wagtailcore_tags wagtailimages_tags homepage_tags wagtailmetadata_tags multi_image_tags i18n static %}
+
+{% block body_id %}youtube-regrets-reporter{% endblock %}
+
+{% block extended_head %}
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Changa" rel="stylesheet">
+{% endblock %}
+
+{% block intro_stationary_wrapper %}
+<div class="dark-theme">
+  <p class="body-large d-none d-lg-block mb-lg-3">{% trans "Download RegretsReporter today." %}</p>
+  <div class="d-flex flex-wrap justify-content-center">
+    <a class="btn btn-primary btn-download btn-firefox-download mt-2 mt-md-0 mx-2" href="https://sites.google.com/mozillafoundation.org/regrets-reporter-staff-test/" target="_blank">{% trans "Add to Firefox" %}</a>
+    <a class="btn btn-primary btn-download btn-chrome-download mt-2 mt-md-0 mx-2" href="https://sites.google.com/mozillafoundation.org/regrets-reporter-staff-test/" target="_blank">{% trans "Add to Chrome" %}</a>
+  </div>
+</div>
+{% endblock %}
+
+{% block main %}
+<div class="container">
+  <div class="row section-how-it-works py-3 py-md-5">
+    <div class="offset-md-2 col-md-8 dark-theme text-center">
+      <h2 class="font-changa">{% trans "How it Works" %}</h2>
+      <p>{% trans "When you send a YouTube Regret, the video and recommendations that led you to it are submitted to Mozilla researchers privately." %}</p>
+    </div>
+    <div class="col-12">
+      <div class="row steps text-center">
+        <div class="col-md-4 mb-3 mb-md-0">
+          <img src="{% static "_images/youtube-regrets/regrets-reporter/how-it-works-step1.png" %}" />
+          <p class="d-flex flex-column align-items-center">
+            <span class="step-number my-3">1</span>
+            {% trans "Click the frowning extension icon in the browser bar" %}
+          </p>
+        </div>
+        <div class="col-md-4 mb-3 mb-md-0">
+          <img src="{% static "_images/youtube-regrets/regrets-reporter/how-it-works-step2.png" %}" />
+          <p class="d-flex flex-column align-items-center">
+            <span class="step-number my-3">2</span>
+            {% if request.LANGUAGE_CODE == "en" %}
+              Report the video and recommendations that led you to it
+            {% else %}
+              {% trans "Report the video* and recommendations that led you to it" %}
+            {% endif %}
```

Comment:
This page is not currently getting localized, due to https://github.com/mozilla/foundation.mozilla.org/pull/5121/files#r482310695, so the string is going to be the same for every user. If not, and that `*` is supposed to be a footnote, that should most definitely be a real link, to the part of the page that explains that footnote. Or even better: not be a footnote at all, since this is a web page and not printed media =)

```suggestion
            {% trans "Report the video and recommendations that led you to it" %}
```

Reply:
updated!